client/component/vite.config.ts

```diff
@@ -1,20 +1,16 @@
-import { defineConfig } from 'vite';
+import { defineConfig, PluginOption } from 'vite';
 import react from '@vitejs/plugin-react';
 import dts from 'vite-plugin-dts';
 import { resolve } from 'path';
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [
-    react(),
-    dts({ rollupTypes: true }),
-  ],
+  plugins: [react(), dts({ rollupTypes: true })] as PluginOption[],
```

Comment:
Is this necessary after updating the package dependencies and reinstalling?

Reply:
The `PluginOption `part?

Reply:
Yes, it looks like it's gone now.